.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
       go: 1.12.x
       script:
         - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1
-        - golangci-lint run --disable errcheck; 
+        - golangci-lint run --disable errcheck --enable golint; 
```

Comment:
Does "--disable errcheck" disable golint?

Reply:
No it just disable the check for error return.

Golint is not enable by default.. so i just included it...